.travis.yml

```diff
@@ -3,9 +3,9 @@ elixir:
   - 1.7
   - 1.8
   - 1.9
+  - 1.10
 
 otp_release:
-  - 20.0
   - 21.0
```

Comment:
I am fine with dropping support for this version. We will have to release a new major version though.

Reply:
Just curious, what are we winning by dropping support for this version?

Reply:
* we can use new features from otp21
* we can continue upgrading `amqp` library, which right now I believe requires otp21 as well

Reply:
The primary reason for the change was so that `handle_continue` could be used to ensure that the init parts are called before anything else in the GenServer.

Reply:
Got it. Thanks!

Reply:
@mkorszun feel free to resolve this conversation, unless you need it as a reminder to "release a new major version".